beaconChain/helpers/stateTransitionHelpers.ts

```diff
@@ -236,7 +243,7 @@ export function getCurrentEpochCommitteeCount(state: BeaconState): int {
  * @returns {Number}
  */
 export function getNextEpochCommitteeCount(state: BeaconState): int {
-  const nextActiveValidators = getActiveValidatorIndices(state.validatorRegistry, getCurrentEpoch(state) + 1);
+  const nextActiveValidators = getActiveValidatorIndices(state.validatorRegistry, getCurrentEpoch(state).addn(1));
```

Comment:
I think this needs to be 
```suggestion
const nextActiveValidators = getActiveValidatorIndices(state.validatorRegistry, getCurrentEpoch(state).addn(new BN(1)));
```